### ISSUE TITLE
Language and connection lost

### ISSUE THREAD

**Issue by franchesko1437** [Levak/warfacebot#91]
1. How can I change the language in a bot?
2. How to throw people out of the game? "Connection lost to server"

**Comment by Levak** [comment]
1. Change `g_language` in wb.cfg for one of the supported languages in `cfg/lang` folder
2. MRAC disconnects bots after 4min, nothing to do about it.

**Comment by franchesko1437** [comment]
1. I changed the folder, but in cmd it did not change. Also remained a slavish and English
Screen: http://s018.radikal.ru/i509/1707/7d/e5e74ac30d0b.jpg

2. I asked, not about why the bot crashes. 
But about what, how. Throw out other people with commands.  Some craftsmen on the Russian server do so. 
Go into the room, and throw out a person with a disconnect

**Comment by Levak** [comment]
1. That's a Windows problem and idk how to fix it. See #66 
 2. I have no info on that, and it's probably not using warfacebot :/
 3. Ask them how they did with the source code, according to AGPLv3 license of warfacebot.

**Comment by franchesko1437** [comment]
1. But I do not know where to put it and pack it from source! Can you Guide me.
![default](https://user-images.githubusercontent.com/29465298/27764446-b170fffa-5eaa-11e7-92d3-37257b9614d7.JPG)
The language has not changed

2.  This is due to sending requests to the client. You know how to use the game code. Must know how to do this

3.They are not connected, and they will not give me the code to restart automatically. Could you tell me how to do this?

**Comment by franchesko1437** [comment]
Levak?????

**Comment by Levak** [comment]
1. As I said, I don't really know how to fix it for now
2. Well, idk either. AFAIK all the flaws that I've found have been fixed after I reported them. And even if I knew one, I would not disclose it, but report it. Asking for it is just boggus coming from you.
3. I probably can, but I won't. This is not how warfacebot was designed to be (reconnect every 2min), so adding support of auto-reconnect won't help. Only part of warfacebot that supports failover is its DBus API, but I doubt it will help you as a Windows user.
